proto/frequenz/api/dispatch/dispatch.proto

```diff
@@ -28,6 +28,13 @@ service DispatchService {
 
   // Update a dispatch
   rpc UpdateDispatch(DispatchUpdateRequest) returns (google.protobuf.Empty);
+
+  // Get a single dispatch
+  rpc GetDispatch(DispatchGetRequest) returns (Dispatch);
+
+  // Delete a given dispatch
+  // Performs a "soft-delete" by marking the given dispatch as deleted in the database.
+  rpc DeleteDispatch(DispatchDeleteRequest) returns (google.protobuf.Empty);
```

Comment:
Would it make sense to return the deleted dispatch entity here?

Reply:
I am not sure. There isn't a standard definition for this in either REST or gRPC. However, I can't really think of a use case where this would be useful, so unless you have one, I think we can leave it empty.